brambling/templates/brambling/event/organizer/custom_form_list.html

```diff
@@ -0,0 +1,33 @@
+{% extends 'brambling/event/organizer/__base.html' %}
+
+{% load zenaida %}
+
+{% block title %}Forms – {{ block.super }}{% endblock %}
+
+{% block main %}
+	{{ block.super }}
+
+	{% include "brambling/event/_empty_shop_alert.html" %}
+
+	<p><a class="btn btn-primary" href="{% url 'brambling_form_create' event_slug=event.slug %}">Create form</a></p>
+	{% if custom_forms %}
+		<table class='table table-striped'>
+			<thead>
+				<tr>
+					<th>Name</th>
+					<th>Type</th>
+					<th>Index</th>
```

Comment:
Index? The forms are displayed in an order... i.e. if there are multiple Order forms, they would be displayed in index order. Someday that will translate to which page they are on.